operator/gitops/argocd/argo-apps/cert-manager-operator.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     server: https://kubernetes.default.svc
   source:
     path: operator/cert-manager
-    repoURL: https://github.com/openshift-pipelines/pipeline-service.git
+    repoURL: https://github.com/AndrienkoAleksandr/pipelines-service.git
```

Comment:
Any reason why you need to test with your fork for cert-manager?

Reply:
No, that's only for case if in the original repo was changed folder location due refactoring or someting like that.